src/registrar/tests/test_views_portfolio.py

```diff
@@ -327,13 +327,36 @@ def test_portfolio_organization_info_page_edit_access(self):
             response = self.app.get(reverse("organization-info"))
             # Assert the response is a 200
             self.assertEqual(response.status_code, 200)
-            # The label for Federal agency will always be a h4
-            self.assertContains(response, '<h4 class="margin-bottom-05">Organization name</h4>')
-            # The read only label for city will be a h4
-            self.assertNotContains(response, '<h4 class="margin-bottom-05">City</h4>')
-            self.assertNotContains(response, '<p class="margin-top-0">Los Angeles</p>')
+            self.assertContains(response, "<h2>Organization admins</h2>")
```

Comment:
Ideally I would also check that the admin names are included, but until we have beautifulsoup to parse the html, I'm deferring that level of granularity. I may work on adding it in a different PR.